src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let price = 40 * days;
```

Comment:
Consider using a constant for the daily rental cost instead of a hardcoded value.